README.md

```diff
@@ -2,12 +2,33 @@
 
 # Usage:
 
-## For Developers:
+To build a distro-specific package (rpm or deb):
 
-Making a developer package is as simple as:
+```bash
+make clean
+make docker.io/library/<distro>:<version> [docker.io/library/<distro>:<version> ...]
+
+# for example:
+# make docker.io/library/centos:7
+# make docker.io/library/ubuntu:18.04
+```
+
+After build completes, packages can be found in the `build` directory.
+
+## Building a package from a local source directory
+
+Specify the path to the local source directory using `CONTAINERD_DIR` and/or
+`RUNC_DIR`:
+
+```shell
```

Comment:
nit: we use `bash`elsewhere

Reply:
I believe `console` and `shell` officially are the "preferred" ones (`bash` being for Bash _scripts_), but we've used `bash` everywhere. Let me change these to use `bash` for now and we can consider updating it to `shell` / `console` across the board separately.